x-pack/solutions/observability/plugins/observability/public/attachments/page/translations.tsx

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const ADDED_PAGE = i18n.translate('xpack.observability.caseView.links.addedLink', {
```

Comment:
naming nit: More of a general bservation... Should this be a `LinkAttachment` or `PageAttachment`? Just want to unify some of the naming references around `addedLink` or `addedPage` etc...